Python/sampleclient/commands.py

```diff
@@ -26,7 +26,7 @@ def parse_args():
         default='AZURE',
         required=True,
         help="""specifies the method of file uplaod for submission of the job. The options are
-                - AZURE: The job will leverage Micrsoft AZURE Binary Blob Storage.
+                - AZURE: The job will leverage Microsoft AZURE Binary Blob Storage.
```

Comment:
I guess, thinking about branding, "Azure" is not an acronym/all caps, so we should probably tweak that.

Reply:
Thanks for your patience, I was out with a Fever for a few days.

I have updated the AZURE to be Azure here.